### DISCUSSION TITLE
[Beta Pass] How to acquire?

### DISCUSSION THREAD

**Discussion by future2back** [sunflower-land/sunflower-land#2019] [General]
What can we do to acquire? Will it be available for players who want to help the project or just for a select few?

I would like to know what I can do to have this pass and how it will be distributed, if it will be considered normal players or just players who have some requirements?

Thank you.

**Comment by IvanSkazkin** [comment]
I support. The same question. Why wasn't the community informed? And only given to certain people? I am in the project 24/7. I would like to participate. I honor myself as deprived. After all, there was not even an announcement and some applications

**Comment by NomiceBinocle** [comment]
it's written in the whitepaper you need a moderator to have the beta pass ....

in my opinion he should provide it to players with the warrior badge too but no interest in providing it to classic players

**Comment by digital-farms** [comment]
> What can we do to acquire? Will it be available for players who want to help the project or just for a select few?

**Beta pass is given to moderators/certified coders/certified artists/community elders**

**Comment by adamhannigan** [comment]
Great question!

We've found that the perfect number of people for beta testing is around 30-40 people. This gives us a large sample size, but also ensures that the amount of feedback is manageable.

We need to keep this group small so we can appropriately deal with the incoming feedback, manage fixes and much more. In the past, when we had a large amount of people it became near impossible to reach consensus and it was just a flood of messages/feedback 😅 

Right now the people chosen for this group are:

1. Contributors, since they are the ones who help fix, design, build and polish the feature
2. Moderators, since they 'represent' a wide range of nationalities + groups in the community - it helps give an unbiased opinion.

There may be special events where other community members can earn a Beta Pass, but we need to be careful how many people can enter this group.

I hope that clarifies it 😄

**Reply by IvanSkazkin** [reply]
**Thanks for answering our questions. This is very valuable!**

**Reply by future2back** [reply]
Hello @adamhannigan thanks for replying.

What worries me is that this sample may not give a real feedback as is expected. The group you mentioned probably are big players, with a lot of boosts. I think this can affect the quality of provided feedback.

What i propose is:

- Do a daily check in of the players.

- Make Beta Pass as temporary title.
  e.g. Betapass#1 - Used to test apples.
         Betapass#2 - Used to test oranges.
   
- All players who have been active for more than 30 days can be eligible for the Beta Pass draw.

- Hold a raffle with a sample defined by the devs.

- Make a  5-10min survey with filters, to make sure the answers make sense to the questioned, if the player fails the filter will not be able to continue the poll until the end. Give a small prize to the player who reaches the end. 
These surveys could appear not only for those who are testing a tool at the moment on Beta Pass, but also to active players to make market research inside game, since players don't need to know the answers developers are looking for, we have a lot of tools to do this and have constant feedback from the active community.

- At the end remove Beta Pass title, tools tested, and all resources extracted under tests automatically on the next sync. I believe this is very important in order not to be unfair by leaving other players behind.


I believe that if an idea like this is implemented, we can get more accurate and real feedback from the community.

Thank you.

**Comment by brian-man** [comment]
> the perfect number of people for beta testing is around 30-40 people

There are 54 beta passes currently, IIRC.

**Reply by Aminbtdds** [reply]
Yes, it was 54 beta passes(actually 52, farm 1 and 2 have 2 each, obviously for testing it).
17 out of 54 have coder badge.
11 out of 54 have artist badge.
20 out of 54 have discord mod badge.
22 out of 54 have liquidity provider badge.
Ofc these groups have overlaps which i didn't check, i didn't liked to be too curious about it, it's a privilage and also honor👌🌻❤
